doc/man3/BIO_ctrl.pod

```diff
@@ -72,6 +74,9 @@ Not all BIOs support these calls. BIO_ctrl_pending() and BIO_ctrl_wpending()
 return a size_t type and are functions, BIO_pending() and BIO_wpending() are
 macros which call BIO_ctrl().
 
+BIO_get_ktls() return 1 if the BIO is using the Kernel TLS data-path for
```

Comment:
Would `BIO_get_ktls_send()` or similar be a better name? Given that we know we are going to need a similar macro for the receive side int he future

Reply:
Done.